test/db/cmd/types

```diff
@@ -119,11 +119,11 @@ EOF
 EXPECT=<<EOF
 | ; var struct JNINativeInterface @ sp+0x12c
 | 0x00000f74      4b93           str r3, [sp + JNINativeInterface]
-| 0x00000f76      039b           ldr r3, [sp + var_128h]               ; JNINativeInterface.reserved0
```

Comment:
slightly unrelated question, but do you know why here the variables is 128h/134h? With `e asm.var.sub=false` I see `ldr r3, [sp, 0xc]`, so seeing `128h` seems strange :/

Reply:
@ret2libc its because of `e anal.vars.stackname=true`